res/values/strings.xml

```diff
@@ -678,7 +697,16 @@
     <string name="menu.view.developer.open.log.folder">Open the log folder</string>
     <string name="menu.view.developer.open.current.log.file">Open current logfile</string>
     <string name="user_location_permission_explanation">Delta Chat needs the location permission in order to show and share your location.</string>
+    <string name="select_profile_image_desktop">Select your new profile image</string>
+    <plurals name="ask_send_file_desktop">
+        <item quantity="one">Send the following file to %s?</item>
+        <item quantity="other">Send the following %d files to %s?</item>
```

Comment:
"this file(s)" instead of "the following file(s)" would be shorter

Reply:
do you mean "these file(s)"?
the context of this is: https://github.com/deltachat/deltachat-desktop/pull/1640
https://user-images.githubusercontent.com/18725968/81891069-d0009e80-95a7-11ea-8858-f7a511f9cd42.png

Reply:
Yes, of course

```suggestion
        <item quantity="one">Send this file to %s?</item>
        <item quantity="other">Send these %d files to %s?</item>
```